### ISSUE TITLE
Not able to find action assets after copy MODULE_NOT_FOUND

### ISSUE THREAD

**Issue by timheuer** [nektos/act#391]
This is a different error than previous MODULE_NOT_FOUND

Environment: windows
Repo for active repro: https://github.com/timheuer/base64-to-file

Results on 'act' -- when looking in the container though there does appear to have it copied
```
[Action Test/test-2] 🚧  Skipping unsupported platform '[${{ matrix.os }}]'
[Action Test/test-1] 🧪  Matrix: map[os:ubuntu-latest]
[Action Test/test-1] 🚀  Start image=node:12.6-buster-slim
[Action Test/test-1]   🐳  docker run image=node:12.6-buster-slim entrypoint=["/usr/bin/tail" "-f" "/dev/null"] cmd=[]
[Action Test/test-1]   🐳  docker cp src=C:\Users\timheuer\Documents\GitHub\base64-to-file\. dst=\github\workspace
[Action Test/test-1] ⭐  Run Checkout
[Action Test/test-1]   ✅  Success - Checkout
[Action Test/test-1] ⭐  Run Run Workflow
| internal/modules/cjs/loader.js:628
|     throw err;
|     ^
|
| Error: Cannot find module '/github/workspace/base64-to-file/dist/index.js'
|     at Function.Module._resolveFilename (internal/modules/cjs/loader.js:625:15)
|     at Function.Module._load (internal/modules/cjs/loader.js:527:27)
|     at Function.Module.runMain (internal/modules/cjs/loader.js:839:10)
|     at internal/main/run_main_module.js:17:11 {
|   code: 'MODULE_NOT_FOUND',
|   requireStack: []
| }
[Action Test/test-1]   ❌  Failure - Run Workflow
```

**Comment by awildeep** [comment]
I am having a similar issue trying to run a fork of https://github.com/actions/typescript-action.

**Comment by CyberAndrii** [comment]
Same after updating to the latest release (v0.2.16). Caused by `- uses: ./`. Fixed by downgrading to the v0.2.15.
